src/components/Tutorial.tsx

```diff
@@ -17,7 +17,11 @@ export default function Tutorial({
       <Toolbar />
       <Box m={2}>
         <main>{tutorialSteps[selectedStep].content}</main>
-        <HiraganaTableDialog></HiraganaTableDialog>
+        {tutorialSteps[selectedStep].hasHintHiraganaTable ? (
+          <HiraganaTableDialog></HiraganaTableDialog>
+        ) : (
+          " "
+        )}
```

Comment:
[NITS]
これでも動くかも:eyes:

```suggestion
        {tutorialSteps[selectedStep].hasHintHiraganaTable && (
          <HiraganaTableDialog />
        )}
```